docker/Dockerfile

```diff
@@ -18,17 +18,18 @@
 # Docker image for apache kylin, based on the Hadoop image
 FROM hadoop2.7-all-in-one
 
-ENV KYLIN_VERSION 3.0.1
+ENV KYLIN_VERSION 3.1.0
 ENV KYLIN_HOME /home/admin/apache-kylin-$KYLIN_VERSION-bin-hbase1x
 
 # Download released Kylin
-RUN wget https://archive.apache.org/dist/kylin/apache-kylin-$KYLIN_VERSION/apache-kylin-$KYLIN_VERSION-bin-hbase1x.tar.gz \
+RUN wget https://downloads.apache.org/kylin/apache-kylin-$KYLIN_VERSION/apache-kylin-$KYLIN_VERSION-bin-hbase1x.tar.gz \
```

Comment:
I may not accpect this change from **archive** to "mirror", older version is not accessible in mirror, like 2.5.0.

Reply:
it can't download kylin 3.1.0 from https://archive.apache.org at that time, so I changed to "mirror".

Reply:
Done